rc/core/python.kak

```diff
@@ -47,19 +47,85 @@ add-highlighter -group /python/comment       fill comment
 
     # Highlight keywords
     printf %s "
-        add-highlighter -group /python/code regex '\b(${values})\b' 0:value
-        add-highlighter -group /python/code regex '\b(${meta})\b' 0:meta
-        add-highlighter -group /python/code regex '\b(${keywords})\b' 0:keyword
+        add-highlighter -group /python/code regex '\b(${values})\b' 1:value
+        add-highlighter -group /python/code regex '\b(${meta})\b' 1:meta
+        add-highlighter -group /python/code regex '\b(${keywords})\b' 1:keyword
         add-highlighter -group /python/code regex '\b(${functions})\b\(' 1:builtin
     "
 
     # Highlight types and attributes
     printf %s "
-        add-highlighter -group /python/code regex '\b(${types})\b' 0:type
+        add-highlighter -group /python/code regex '\b(${types})\b' 1:type
         add-highlighter -group /python/code regex '@[\w_]+\b' 0:attribute
     "
 }
 
+# extended syntax
+
+%sh{
+    values="self"
+    # curl -s "https://docs.python.org/3/reference/datamodel.html#special-method-names" | ag -o '__[a-z_]+__[^(]' | sed s'/.$//' | sort | uniq
+    attributes="__abs__|__add__|__aenter__|__aexit__|__aiter__|__and__|__anext__"
+    attributes="${attributes}|__annotations__|__await__|__bases__|__bool__|__bytes__"
+    attributes="${attributes}|__call__|__class__|__classcell__|__closure__|__code__"
+    attributes="${attributes}|__complex__|__contains__|__defaults__|__del__"
+    attributes="${attributes}|__delattr__|__delete__|__delitem__|__dict__|__dir__"
+    attributes="${attributes}|__divmod__|__doc__|__enter__|__eq__|__exit__|__file__"
+    attributes="${attributes}|__float__|__floordiv__|__format__|__func__|__future__"
+    attributes="${attributes}|__ge__|__get__|__getattr__|__getattribute__"
```

Comment:
I don't understand the distinction between what you have called `attributes` and  what you have called `methods`. Some of them are listed in both for instance `__ge__` here...